Objects/abstract.c

```diff
@@ -2627,7 +2627,7 @@ recursive_issubclass(PyObject *derived, PyObject *cls)
     int is_union = (PyType_Check(type) && type == &_Py_UnionType);
     if (!is_union && !check_class(cls,
                             "issubclass() arg 2 must be a class,"
-                            " a tuple of classes, or a union.")) {
+                            " a tuple of classes or a union")) {
```

Comment:
The comma before "or" is an Oxford comma and is correct for this use. It improves clarity that "or a union" refers to "arg 2" and not to "a tuple of".

Let's keep the Oxford comma. Please add it to the other error message for consistency.

Reply:
OK.